hca2scea-backend/script_spreadsheets/DevelopingCardiacSystem_test/updated_project_details.json

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
do we need this file in github?

Reply:
Nope! I intend to modify the script to update this file automatically too so it won't be needed. I just wanted to keep it for the moment to facilitate testing.